plugin/route53/setup.go

```diff
@@ -31,6 +31,22 @@ var f = func(credential *credentials.Credentials) route53iface.Route53API {
 	return route53.New(session.Must(session.NewSession(&aws.Config{Credentials: credential})))
 }
 
+func parseDurationFromCaddyController(cc *caddy.Controller, def time.Duration) (duration time.Duration, err error) {
+	var durationStr string = cc.Val()
+	_, err = strconv.Atoi(cc.Val())
```

Comment:
why? Just call ParseDuration

Reply:
If we're keeping this config knob in the first place; which I think we shouldn't

Reply:
reverted the original file.